rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -5,110 +5,80 @@
-            }
-    ),
-    COMMIT_FEDERATION("commit_federation",
-            new CallTransaction.Param[]{
-                    new CallTransaction.Param(false, "oldFederationBtcPublicKeys", SolidityType.getType(SolidityType.BYTES)),
-                    new CallTransaction.Param(false, "oldFederationBtcAddress", SolidityType.getType(SolidityType.STRING)),
-                    new CallTransaction.Param(false, "newFederationBtcPublicKeys", SolidityType.getType(SolidityType.BYTES)),
-                    new CallTransaction.Param(false, "newFederationBtcAddress", SolidityType.getType(SolidityType.STRING)),
-                    new CallTransaction.Param(false, "activationHeight", SolidityType.getType(SolidityType.INT256))
-            }
-    ),
-    RELEASE_REQUESTED("release_requested",
-            new CallTransaction.Param[]{
-                    new CallTransaction.Param(true, "rskTxHash", SolidityType.getType(SolidityType.BYTES32)),
-                    new CallTransaction.Param(true, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
-                    new CallTransaction.Param(false, Fields.AMOUNT, SolidityType.getType(SolidityType.UINT256))
-            }
-    ),
-    RELEASE_REQUEST_RECEIVED_LEGACY("release_request_received",
-            new CallTransaction.Param[]{
-                    new CallTransaction.Param(true, Fields.SENDER, SolidityType.getType(SolidityType.ADDRESS)),
-                    new CallTransaction.Param(false, "btcDestinationAddress", SolidityType.getType(SolidityType.BYTES)),
-                    new CallTransaction.Param(false, Fields.AMOUNT, SolidityType.getType(SolidityType.UINT256))
-            }
-    ),
-    RELEASE_REQUEST_REJECTED("release_request_rejected",
-        new CallTransaction.Param[]{
-            new CallTransaction.Param(true, Fields.SENDER, SolidityType.getType(SolidityType.ADDRESS)),
-            new CallTransaction.Param(false, Fields.AMOUNT, SolidityType.getType(SolidityType.UINT256)),
-            new CallTransaction.Param(false, Fields.REASON, SolidityType.getType(SolidityType.INT256))
-        }
-    ),
-    BATCH_PEGOUT_CREATED("batch_pegout_created",
-        new CallTransaction.Param[]{
-            new CallTransaction.Param(true, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
-            new CallTransaction.Param(false, Fields.RELEASE_RSK_TX_HASHES, SolidityType.getType(SolidityType.BYTES))
-        }
-    ),
-    RELEASE_REQUEST_RECEIVED("release_request_received",
-                                            new CallTransaction.Param[]{
+    LOCK_BTC("lock_btc", new CallTransaction.Param[] {
+        new CallTransaction.Param(true, Fields.RECEIVER, SolidityType.getType(SolidityType.ADDRESS)),
+        new CallTransaction.Param(false, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
+        new CallTransaction.Param(false, "senderBtcAddress", SolidityType.getType(SolidityType.STRING)),
```

Comment:
No big deal, but, why do we keep a mixture of `Fields` constants and literal strings? Should we always use `Fields` constants instead?

Reply:
No particular reason to be honest, it's just the way it has evolved